packages/client/machines/appMachine.ts

```diff
@@ -524,6 +524,36 @@ export function createAppMachine({
                                     },
                                 },
                             },
+
+                            /**
```

Comment:
Question: Is the user able to go back on every reset password wizard form screen ? event the set new password one ?

Reply:
Yes, the user can go back from any screen about password reset.